UnitTests/SqlPreparation/SqlPreparationQueryModelVisitorTest.cs

```diff
@@ -405,25 +402,24 @@ public void VisitWhereClause_MulipleWhereClauses ()
       var preparedExpression2 = Expression.Constant (false);
 
       _stageMock
-          .Expect (
-              mock =>
-              mock.PrepareWhereExpression (
-                  Arg<Expression>.Matches (e => e == predicate1),
-                  Arg<ISqlPreparationContext>.Matches (c => c != _context)))
-          .Return (preparedExpression1);
+          .Setup (
+              mock => mock.PrepareWhereExpression (
+                  predicate1,
```

Comment:
don't we need to use It.Is for expression?

Reply:
Docs for `ItExpr` state:

> Allows the specification of a matching condition for an argument in a protected member setup, rather than a specific argument value. "ItExpr" refers to the argument being matched.

My interpretation is it is only used as an `It.Is` replacement when mocking protected members like

```
handlerMock
  .Protected()
  .Setup<Task<HttpResponseMessage>>(
    "SendAsync",
    ItExpr.IsAny<HttpRequestMessage>(),
    ItExpr.IsAny<CancellationToken>())
```